CHANGELOG.md

```diff
@@ -21,6 +21,9 @@
   PR [#2561](https://github.com/realm/realm-core/pull/2561).
 * New replication instruction: instr_AddRowWithKey
 * Add the old table size to the instr_TableClear replication instruction.
+* Throw a MaxAddressBreached exception in some places instead of wrapping around.
```

Comment:
Which places this is thrown has to be spec'ed, so bindings can figure out where to check for this.
"Instead of wrapping around"? Should that have been instead of asserting? Instead of crashing?
How must the bindings react when this exception is thrown?